### PR TITLE
Add test cases to verify ca-AD (Catalan-Andorra) and ca-ES(Catalan-Spain)

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Removed duplicated locale data 
  * Manually confirmed with cldr 36 data 
  * no difference between ca-AD and ca-ES 
* Added testcase to verify ca-AD and ca-ES
* Added test case feature: LocaleInfo,Script, String(plural) LocaleMatch, DateFmt, DurFmt, DateRngFmt, List, Unit, firstDayofWeek, clock, calendar,Month/Weekday Translation, NumFmt,

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/test/root/testscriptinfo.js [hunk 1]
```diff
@@ -2185,5 +2185,25 @@ module.exports.testscriptinfo = {
         test.equal(li.getScript(), "Geor");
         test.equal(scinfo.getScriptDirection(), "ltr");
         test.done();
+    },
+    testScriptInfo_es_AD: function(test) {
+        test.expect(4);
+        var li = new LocaleInfo("es-AD");
```
Comment (ehoogerbeets) [inline js/test/root/testscriptinfo.js:2191]: Should be ca-AD

File: js/test/root/testscriptinfo.js [hunk 2]
```diff
@@ -2185,5 +2185,25 @@ module.exports.testscriptinfo = {
...
+        var scinfo = new ScriptInfo(li.getScript());
+        test.ok(li !== null);
+        test.ok(scinfo !== null);
+        test.equal(li.getScript(), "Latn");
+        test.equal(scinfo.getScriptDirection(), "ltr");
+        test.done();
+    },
+    testScriptInfo_es_ES: function(test) {
+        test.expect(4);
+        var li = new LocaleInfo("es-ES");
```
Comment (ehoogerbeets) [inline js/test/root/testscriptinfo.js:2201]: Should be ca-ES

File: js/test/root/teststrings.js [hunk 3]
```diff
@@ -3670,5 +3691,25 @@ module.exports.teststrings = {
 
         test.equal(str.formatChoice(1.7), "Default items");
         test.done();
+    },
+    testStringFormatChoiceDecimal_ca_AD: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("ca-AD");
```
Comment (ehoogerbeets) [inline js/test/root/teststrings.js:3698]: Should this be ca-ES?

File: js/test/root/teststrings.js [hunk 4]
```diff
@@ -3670,5 +3691,25 @@ module.exports.teststrings = {
...
+
+        test.ok(str !== null);
+
+        test.equal(str.formatChoice(1.7), "Default items");
+        test.done();
+    },
+    testStringFormatChoiceDecimal_ca_AD2: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
+        str.setLocale("ca-AD");
```
Comment (ehoogerbeets) [inline js/test/root/teststrings.js:3708]: And this one ca-ES too?

**Comment by ehoogerbeets** [comment]
Also, please update the year in the copyright notice for each file you changed to include 2020.

File: js/test/root/testscriptinfo.js (hunk 1 above)
Comment (gouniLee) [reply js/test/root/testscriptinfo.js]: oh yes.. I fixed it.

File: js/test/root/testscriptinfo.js (hunk 2 above)
Comment (gouniLee) [reply js/test/root/testscriptinfo.js]: I fixed it.

File: js/test/root/teststrings.js (hunk 3 above)
Comment (gouniLee) [reply js/test/root/teststrings.js:3698]: This is correct. I additional added test cases for ca-ES

File: js/test/root/teststrings.js (hunk 4 above)
Comment (gouniLee) [reply js/test/root/teststrings.js:3708]: I newly added test cases for ca-ES

**Comment by gouniLee** [comment]
> Also, please update the year in the copyright notice for each file you changed to include 2020.

**Comment by gouniLee** [comment]
I accidentally clicked the `close` button. ;(..  so I opened again.  Regarding a copyright year, It looks like I put 2020 all, and some of that has already included 2020. ( Actually All of the separated PR that I made, touching the same files. so The latest PR include might have updated copyright year even though the old PR haven't updated)

**Pull request merged**